Map Version Edits.R

```diff
@@ -217,30 +243,35 @@ dan_func_single_obs <- function(x, parcels_full, parcels_buffered, network) {
     st_transform(3435)
 
 
+  # Step 5: Get the number of roads that are bordering the parcel
   touching_unit_street <- rep(list(NULL), 4)
-  
+
   touching_unit_street <- suppressWarnings(suppressMessages(
     st_intersects(cross_filter$geometry, network)
   ))
-  
+
   touching_unit_street <- c(touching_unit_street, rep(list(NULL), 4 - length(touching_unit_street)))
-  
+
   touching_street_number <- sum(lengths(touching_unit_street))
-  
+
   cross_corner_number <- nrow(cross_corner)
-  
+
   aspect_ratio <- max(cross$aspect_ratio, na.rm = TRUE)
-  
+
+  # Check for an edge case where two corner segments are actually on opposing
+  # sides of the parcel
```

Comment:
To clarify my understanding: Is the following block attempting to accomplish roughly the same thing as the angle filter above?

Reply:
I'm pretty sure that the aspect ratio is the ratio for the length of two sides of a parcel, that the length is not significantly longer than the width.

Reply:
I meant to reference this block in my comment:

https://github.com/ccao-data/data-architecture/blob/0f2c63a33677e5ccc8b210d811c8c3380e80655b/Map%20Version%20Edits.R#L225-L245

Which seems different from the aspect ratio calculation:

https://github.com/ccao-data/data-architecture/blob/0f2c63a33677e5ccc8b210d811c8c3380e80655b/Map%20Version%20Edits.R#L186-L197

But I think your response here answers my question! https://github.com/ccao-data/data-architecture/pull/260#discussion_r1414368160